src/frontend/components/registrationForm.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from "react";
+import Input from "./input";
+import Button from "./button";
+import Select from "./Select";
+
+class RegistrationForm extends Component {
+  constructor(props) {
+    super(props);
+  }
+  handleChange = () => {
+    console.log("Changes made in input ");
+  };
+  handleSubmit = () => {
+    console.log("Submit button clicked");
+  };
+  getDays = () => {
+    const days = [];
+    for (let i = 1; i <= 31; i++) {
+      days.push(i);
+    }
+    return days;
+  };
+  getMonths = () => {
+    var months = [
+      "Jan",
+      "Feb",
+      "Mar",
+      "Apr",
+      "May",
+      "Jun",
+      "Jul",
+      "Aug",
+      "Sep",
+      "Oct",
+      "Nov",
+      "Dec"
+    ];
+    return months;
+  };
+  getYear = () => {
+    const years = [];
+    const currentYear = new Date().getFullYear();
+    for (let i = 1920; i <= currentYear; i++) {
```

Comment:
This should be reversed in my opinion. It would be helpful if users see the current year first. Good logic for generating the years by the way.